pkg/search/postgres/common.go

```diff
@@ -1038,6 +1038,9 @@ func RunCursorQueryForSchemaFn[T any, PT pgutils.Unmarshaler[T]](ctx context.Con
 
 		var data []byte
 		tag, err := pgx.ForEachRow(rows, []any{&data}, func() error {
+			if ctx.Err() != nil {
```

Comment:
verify we don't need to do anything with `rows` when we exit.  I can never remember when we need to do something or don't.

Reply:
`pgx.ForEachRow` closes them